compose.yaml

```diff
@@ -1,12 +1,12 @@
 version: '3'
 services:
   raphael_frontend_react:
-    image: localhost/raphael/frontend
+    image: ${REGISTRY}fullfact/raphael-frontend:latest
```

Comment:
If `REGISTRY` is unset, it’ll just be left blank, so when running locally the image will be at: `fullfact/raphael-frontend:latest`

Reply:
I _think_ this would work on the server even without `REGISTRY` being set because docker likes to default to `docker.io` by default if it doesn't find the image locally, however it's better to be explicit like this.

Reply:
Nice! Thank you!